instructionAPI/src/AMDGPU/gfx908/finalizeOperands.C

```diff
@@ -1,2892 +1,2265 @@
-//V_CMP_F_F64,V_CMP_LT_F64,V_CMP_EQ_F64,V_CMP_LE_F64,V_CMP_GT_F64,V_CMP_LG_F64,V_CMP_GE_F64,V_CMP_O_F64,V_CMP_U_F64,V_CMP_NGE_F64,V_CMP_NLG_F64,V_CMP_NGT_F64,V_CMP_NLE_F64,V_CMP_NEQ_F64,V_CMP_NLT_F64,V_CMP_TRU_F64,V_CMP_F_I64,V_CMP_LT_I64,V_CMP_EQ_I64,V_CMP_LE_I64,V_CMP_GT_I64,V_CMP_NE_I64,V_CMP_GE_I64,V_CMP_T_I64,V_CMP_F_U64,V_CMP_LT_U64,V_CMP_EQ_U64,V_CMP_LE_U64,V_CMP_GT_U64,V_CMP_NE_U64,V_CMP_GE_U64,V_CMP_T_U64
-insn_in_progress->appendOperand(decodeOPR_VCC(0,64),false,true);
-insn_in_progress->appendOperand(decodeOPR_SRC(layout.SRC0+0,2),true,false);
-insn_in_progress->appendOperand(decodeOPR_SRC(layout.SRC0+1,0),true,false);
-insn_in_progress->appendOperand(decodeOPR_VGPR(layout.VSRC1+0,2),true,false);
-insn_in_progress->appendOperand(decodeOPR_VGPR(layout.VSRC1+1,0),true,false);
-break;
-case 112:case 113:case 114:case 115:case 116:case 117:case 118:case 119:case 120:case 121:case 122:case 123:case 124:case 125:case 126:case 127:case 240:case 241:case 242:case 243:case 244:case 245:case 246:case 247:case 248:case 249:case 250:case 251:case 252:case 253:case 254:case 255:
-//V_CMPX_F_F64,V_CMPX_LT_F64,V_CMPX_EQ_F64,V_CMPX_LE_F64,V_CMPX_GT_F64,V_CMPX_LG_F64,V_CMPX_GE_F64,V_CMPX_O_F64,V_CMPX_U_F64,V_CMPX_NGE_F64,V_CMPX_NLG_F64,V_CMPX_NGT_F64,V_CMPX_NLE_F64,V_CMPX_NEQ_F64,V_CMPX_NLT_F64,V_CMPX_TRU_F64,V_CMPX_F_I64,V_CMPX_LT_I64,V_CMPX_EQ_I64,V_CMPX_LE_I64,V_CMPX_GT_I64,V_CMPX_NE_I64,V_CMPX_GE_I64,V_CMPX_T_I64,V_CMPX_F_U64,V_CMPX_LT_U64,V_CMPX_EQ_U64,V_CMPX_LE_U64,V_CMPX_GT_U64,V_CMPX_NE_U64,V_CMPX_GE_U64,V_CMPX_T_U64
-insn_in_progress->appendOperand(decodeOPR_VCC(0,64),false,true);
-insn_in_progress->appendOperand(decodeOPR_SRC(layout.SRC0+0,2),true,false);
-insn_in_progress->appendOperand(decodeOPR_SRC(layout.SRC0+1,0),true,false);
-insn_in_progress->appendOperand(decodeOPR_VGPR(layout.VSRC1+0,2),true,false);
-insn_in_progress->appendOperand(decodeOPR_VGPR(layout.VSRC1+1,0),true,false);
-insn_in_progress->appendOperand(decodeOPR_SDST_EXEC(126),false,true,true);
-break;
-}
+#include "registers/AMDGPU/amdgpu_gfx908_regs.h"
+#include "InstructionDecoder-amdgpu-gfx908.h"
+namespace Dyninst {
+    namespace InstructionAPI {
+        void InstructionDecoder_amdgpu_gfx908::finalizeENC_DSOperands(){
+            layout_ENC_DS & layout = insn_layout.ENC_DS;
+            switch (layout.OP) {
+                case 0:case 1:case 2:case 3:case 4:case 5:case 6:case 7:case 8:case 9:case 10:case 11:case 13:case 18:case 19:case 21:case 30:case 31:case 84:case 85:
+                //DS_ADD_U32,DS_SUB_U32,DS_RSUB_U32,DS_INC_U32,DS_DEC_U32,DS_MIN_I32,DS_MAX_I32,DS_MIN_U32,DS_MAX_U32,DS_AND_B32,DS_OR_B32,DS_XOR_B32,DS_WRITE_B32,DS_MIN_F32,DS_MAX_F32,DS_ADD_F32,DS_WRITE_B8,DS_WRITE_B16,DS_WRITE_B8_D16_HI,DS_WRITE_B16_D16_HI
```

Comment:
There are too many case labels on a single line.  Put a small number and put the comment for those case labels at the end of the line (base the number on a total line length of less than 100 to 120.

```
              case 0:  case 1:  case 2:  // DS_ADD_U32, DS_SUB_U32, DS_RSUB_U32
              case 3:  case 4:  case 5:  // DS_INC_U32, DS_DEC_U32, DS_MIN_I32
```